.travis.yml

```diff
@@ -3,3 +3,17 @@ language: node_js
 script:
     - script/typecheck
     - script/test
+
+addons:
+    sauce_connect: true
+
+env:
+  global:
+    # Sauce Labs credentials
+    - secure: tDG5SKRDTqN5R+zIRT57nUd4wriyc+IveUBjtuo3yaF3vFFkNNMGjT3lwGYSwx0vjHKiLZB0LCSwvg0l8irRME2WwtmHIuA5ItZEqNuyALucrY3bWqcWxJ/jnHeEeOmYt5PXB5NreCWdVojH7udMH7CHBIj+JdD0344FyYzEsVx7ixsdoOiZUa1ZMsgBZFY61oJxoagh62lVszTnD7OqG3Kv0/TnJKvPGAGYp37f1Pmohe84f+B6/E0Ygt7vxDeXHX3I3bwl3a1oAXtDgCY8RzPGUf7lEWNXP22QRW4lQsXuBDIXeD71RscTO7bZBUU5/+3LSeNGri1tOnnsYmRsGfr3oXwFkycJvXGRVWQKgcpZfVacFhuHB9A4Wm3HhD2NTvyPlSucuZ9Lzx+k/KhqREM4yG7s8m7+mMc6C2FH5E5rpFnXPFVIIoJ6lOO64bNZv0J6sGympZEbmdCzZkNly7uR5IHPv/Tw3mmvvwRfHIun02X/DbJj7U8SmBE1Qfps5GFyIRxhd15mM+sHQSSAteimsEffPM8MAV8x37f4xXK1RBxXGdQ/HEkL4oSxAN5YV++FrsqaHesiDjg9K5K9QKwcFBOWskgNqchVLZMK+IEmRduXXzqrcyxopEXIeCUX2s4jBGV/pKui9Cqa7+LSova3HNwso3q6NM9bZD0uc8Q=
+    - secure: R5FRMM+fWZCdCQpWYvpAAxgV9S8VqPWQNI6dHJxtu5AhdEk1dX8osxOlTP0Sfrm0vdfT856cCdxWzTaMDM9OdTqEIXrVjd+Swp2T6ioarRkGaAa9kfIYj94Oz/vomwKoUZOOvjx2oKNvgL4lU8oDo9u0BJHQzFGKQTBcs5eSOM9rSRhfH9obkkQPZhu+lOFmtMmInreSacwE3+YHKRoQDdKfTJQHTSwB2G8nKmOVykspYAhL0PpxBdoJGMZaCheg1o9lilsI/fJUQRI9tAzNlCvzFou5KXxeYAq9p+S2RlCIeV3PdZtV6DhUw1v7mO1O4OiMQyYxrYEVBdCscDA455HFyswmDFlWhHsHXALxIfnOd9J5V09GRSzhZ6Ky4HZNMNOK1QVtYSraxFBHFBQg1Wjoxv3tmKUmIJM47vq9aBQLoPsRv02e7XMvzECjyOQP+JAmGhd+z4IrNGAE+ystl36pCVEZiUvIxnMwbZoKiiHB7tNOzoMkUyp/RxNz6ij/fZYF0BGRM09Kkf+aFwPJkMPeJl67ZJcHfJY1c3oPj+K7ak6NfPPbEmr4mMScBuulByHau+0Wk91h/KG1oKNLrRdUcV4TzEhNPGIbZs8Qzm+d0W18qIL4IPG0n4CSAptmTlTgS+dcFkkQi8NnCq7Y37N/U2QqRZEVmhoeI8FYlYE=
+    - SELENIUM_REMOTE_URL="http://localhost:4445/wd/hub"
+
+  matrix:
+    - SELENIUM_BROWSER=firefox
```

Comment:
I agree. Let's collate a list of what we want. Do we want to emulate specific Android devices, i.e. Galaxy S3 or just generic android:4.4, generic iphone:8 ?